style.css

```diff
@@ -100,21 +96,19 @@ main {
   padding: 1rem;
 }
 
-.contact {
+.Greetings p {
   font-weight: 600;
   font-size: 15px;
   line-height: 24px;
   color: #7f8cff;
-  padding: 1.5rem;
+  margin-top: 6rem;
+  padding-left: 1rem;
 }
 
 .social-icons {
   display: flex;
   gap: 1rem;
-  padding: 0 1.5rem;
   align-items: center;
-  margin-bottom: 15rem;
-  padding-top: 10px;
 }
```

Comment:
**[OPTIONAL]**

- Great job making the adjustment as requested by @Yerimah, however, your change is close but got some more adjustments. see the image below and try fixing the issues with the social media icons that are not well aligned.


![Screenshot from 2022-12-09 20-03-17](https://user-images.githubusercontent.com/59397159/206778746-aa33886d-9349-48ec-bce6-e1a46b930444.png)